src/index.ts

```diff
@@ -1,11 +1,58 @@
 #!/usr/bin/env node --es-module-specifier-resolution=node
-
 import chalk from 'chalk'
 import figlet from 'figlet'
 import clear from 'clear'
 import { Command } from 'commander'
+import { loadProcesses } from './lib/process/index.js'
 
 const program = new Command()
+const example: string = JSON.stringify([{
+  name: 'A test',
+  version: 1,
+  rawRestrictions: [
+    { SenderOwnsAllInputs: [] },
+    { SenderHasInputRole: [
+      {
+        index: 0,
+        roleKey: 'Supplier',
+      },
+    ]},
+    { op: 'and'},
+    { FixedOutputMetadataValueType: [
+      {
+        index: 0,
+        metadataKey: 'SomeMetadataKey',
+        metadataValueType: 'Literal',
+      },
+      {
+        index: 0,
+        metadataKey: 'SomeOtherMetadataKey',
+        metadataValueType: 'File',
+      },
+    ]},
+    { op: 'and'},
+    { op: 'and'},
```

Comment:
I think there's one too many `{ op: 'and'}` in the example. It doesn't look like valid postfix notation because there's an equal number of `restrictions` and `ops`, whereas the number of  `op` should always be one fewer than the number of `restrictions`